external/dockerfile_functions.sh

```diff
@@ -212,6 +224,9 @@ print_ant_install() {
     local file=$1
     local ant_version=$2
     local os=$3
+    if isExternalImageEnabled ; then
+        os=$(getImageOs)
```

Comment:
on line 512 this `os` variable is in Uppercase - which one should it be?

Reply:
They have nothing in common.  The OS was introduced  4 years ago, os was introduced 2 years ago. They both are local. I handle the same additional logic to both of them, as both share the value of current os. I can refactor to be both lowercase (As are other local variables). But I do not strong preffernece, nor do I want to add more complexity to this PR. Still Sharp eye you have. TY!

Both probably share that theirs authors are originally form Asia origin :)